draft-zsf.md

```diff
@@ -0,0 +1,154 @@
+```
+ZIP: 
+Title: Establish the Zcash Sustainability Fund on the Protocol Level
+Owners: Jason McGee <jason@shieldedlabs.com>
+        Mark Henderson <mark@equilibrium.co>
+        Tomek Piotrowski <tomek@eiger.co>
+        Mariusz Pilarek <mariusz@eiger.co>
+Original-Authors: Nathan Wilcox
+Credits: Nathan Wilcox
+         Mark Henderson
+         Jason McGee
+         Tomek Piotrowski
+         Mariusz Pilarek
+Status: Draft
+Category: Ecosystem
+Created: 2023-08-
+License: BSD-2-Clause
+```
+
+# Terminology
+
+The key words "MUST", "SHOULD", "SHOULD NOT", "MAY", "RECOMMENDED", "OPTIONAL", and "REQUIRED" in this document are to be interpreted as described in RFC 2119. [1]
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200. [2]
+
+The term "Block Rewards" refers to the algorithmic
+issuance of ZEC to every block's creator -- part of the consensus rules.
+
+"Issuance" - The method by which unmined or unissued ZEC is converted to ZEC available to users of the network
+
+"We" - the ZIP authors, owners listed in the above front matter
+
+"`MAX_MONEY`" is the ZEC supply cap. For simplicity, this ZIP defines it to be `21,000,000 ZEC`, although this is slightly larger than the actual supply cap of the original ZEC issuance mechanism.
+
+# Abstract 
+
+This ZIP describes the motivation, the necessary changes for, and the implementation specifications for the Zcash Sustainability Fund (ZSF). The ZSF is a proposed alteration to the block rewards system and accounting of unmined ZEC that allows for other sources of funding besides unissued ZEC. This new mechanism for deposits -- that new applications or protocol designs can use to strengthen the long-term sustainability of the network  -- will likely be an important step for future economic upgrades, such as a transition to Proof of Stake or Zcash Shielded Assets, and other potential protocol fees and user applications.
+
+The changes in this ZIP are ultimately minimal, only requiring for the node to track state in the form of a `ZSF_BALANCE`, and for a new transaction field to be added, called `ZSF_DEPOSIT`. While wallet developer would be encouraged to add the `ZSF_DEPOSIT` field to their UIs, no changes or new behavior are absolutely required for developers or ZEC holders.
+
+# Motivation
+
+The Zcash network's operation and development relies fundamentally on the block reward system inherited from Bitcoin. This system currently looks sometihng like this:
+
+- At Every New Block:
+    - Miner rewarded via unissued ZEC
+    - Transaction fees `(inputs - outputs)`
+
+The Zcash Sustainability Fund is a proposed replacement to that payout mechanism, with the relevant parts in *bold* below:
+
+- **Unmined ZEC is now accounted for as `ZSF_BALANCE`**
+- **Transaction includes optional contributions to ZSF via a `ZSF_DEPOSIT` field**
+- Thus, at Every New Block:
+    - Miner still rewarded **from `ZSF_BALANCE`**
```

Comment:
We think it would be helpful to clearly specify that issuance doesn't change in this ZIP.

```suggestion
This ZIP describes the motivation, the necessary changes for, and the implementation specifications for the Zcash Sustainability Fund (ZSF). The ZSF is a proposed alteration to the block rewards system and accounting of unmined ZEC that allows for other sources of funding besides unissued ZEC. This new mechanism for deposits -- that new applications or protocol designs can use to strengthen the long-term sustainability of the network  -- will likely be an important step for future economic upgrades, such as a transition to Proof of Stake or Zcash Shielded Assets, and other potential protocol fees and user applications.

The changes in this ZIP are ultimately minimal, only requiring for the node to track state in the form of a `ZSF_BALANCE`, and for a new transaction field to be added, called `ZSF_DEPOSIT`. While wallet developer would be encouraged to add the `ZSF_DEPOSIT` field to their UIs, no changes or new behavior are absolutely required for developers or ZEC holders.

This ZIP does not change the current ZEC issuance schedule. Any additional amounts in the sustainability fund are reserved for use in future ZIPs.

# Motivation

The Zcash network's operation and development relies fundamentally on the block reward system inherited from Bitcoin. This system currently looks sometihng like this:

- At Every New Block:
    - Miner and funding streams rewarded a constant amount via unissued ZEC (this constant amount halves at specified heights)
    - Transaction fees `(inputs - outputs)`

The Zcash Sustainability Fund is a proposed replacement to that payout mechanism, with the relevant parts in *bold* below:

- **Unmined ZEC is now accounted for as `ZSF_BALANCE`**
- **Transaction includes optional contributions to ZSF via a `ZSF_DEPOSIT` field**
- Thus, at Every New Block:
    - Miner and funding streams rewarded the same constant amount, **but from `ZSF_BALANCE`** (this constant amount still halves at specified heights)
```